src/components/Thought.tsx

```diff
@@ -106,6 +107,7 @@ interface ThoughtProps {
   simplePath: SimplePath
   view?: string | null
   editing?: boolean | null
+  space?: number
```

Comment:
Could you explain the new `space` prop for me?

Reply:
We are passing down the `space` prop (maybe needs a better name), so we can access it on the `ContentEditable` component in order to adjust the width of the `WebView` to prevent text overflow on long thoughts.

Reply:
it's the result of `getThoughtSpacing(actualDistance, depth) * depth` that helps us get the space value that needs to be subtracted from the `WebView` container.

Reply:
We did it this way because we aren't able to globally style a component.

Reply:
Okay, I see. Let's name it something like `marginLeftParent` and provide a descriptive comment to make it clearer.